impl/src/planning/subplanners.rs

```diff
@@ -101,39 +101,30 @@ pub struct WorkspaceSubplanner<'planner> {
   pub(super) settings: &'planner RazeSettings,
   pub(super) platform_details: &'planner Option<PlatformDetails>,
   pub(super) crate_catalog: &'planner CrateCatalog,
-  pub(super) files: &'planner CargoWorkspaceFiles,
-  pub(super) binary_dependencies: &'planner Vec<CrateCatalog>,
-  pub(super) binary_deps_files: &'planner HashMap<String, CargoWorkspaceFiles>,
+  pub(super) metadata: &'planner RazeMetadata,
 }
 
 impl<'planner> WorkspaceSubplanner<'planner> {
   /** Produces a planned build using internal state. */
   pub fn produce_planned_build(&self) -> Result<PlannedBuild> {
+    // Check for errors ---
```

Comment:
This "---" is not a convention of comment writing, as far as I know. I prefer regular comments and avoiding interstitial newlines to communicate that a comment refers to a whole block of code.

Reply:
Updated